src/core/Dropdown/Dropdown.baseStyles.tsx

```diff
@@ -45,51 +45,62 @@ export const baseStyles = withSuomifiTheme(
 
 export const listboxPopoverStyles = withSuomifiTheme(
   ({ theme }: TokensAndTheme) => css`
```

Comment:
What if you would destructure alwaysShowVisualPlaceholder here and use it to conditionally place styles on active items?

Reply:
Then there would be no need for additional template logic or classes.

Reply:
Could do it that way as well I guess. Is there a benefit compared to the class approach? The class approach is used in most places in the library.

Reply:
I guess the end result is the same no matter how it's done, but after discussion I would also stick to this more pure class based approach.